pkgs/tools/text/mdbook-alerts/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, CoreServices
```

Comment:
pass in `darwin` directly to avoid the `inherit`

Reply:
Thank you! I was trying to decipher the error message, about how to use the by-name instead of all packages, when called with an inherit

Reply:
yeah I believe `inherit`-ing in the top-level causes issues with cross-comp as well, per #204303